bs/src/tablecloth.ml

```diff
@@ -432,28 +432,46 @@ module Tuple2 = struct
   
   let second ((_, b) : 'a * 'b) : 'b = b
 
+  let map ~(f : 'b -> 'y) ((a, b) : 'a * 'b) : 'a * 'y = (a, f b)
```

Comment:
Why does this work only on the second argument? If it's from upstream, do you know their justification? Might be worth looking at what Jane Street does for this in Base.

Reply:
I considered not including map here (and also not in Tuple3).

Elm takes a lot of inspiration from Haskell here: https://stackoverflow.com/questions/13442544/why-does-the-2-tuple-functor-instance-only-apply-the-function-to-the-second-elem?noredirect=1&lq=1

The janestreet libs don't define it.

I am in favour of removing it

Reply:
👍